br/tests/lightning_checkonly/data_parquet/export_info_ci.json

```diff
@@ -0,0 +1,16 @@
+{
+  "exportTaskIdentifier": "ci",
```

Comment:
Since this is not a test case aiming at testing aurora snapshot, I suggest rename the parquet file the standard `db.table.seq.parquet` pattern and remove these useless files.